releases/vhd-notes/aks-ubuntu-1604-201903:2019.03.05.txt

```diff
@@ -0,0 +1,113 @@
+Starting build on Tue Mar 5 17:41:16 UTC 2019
```

Comment:
This will need a bit more work for making the notes formatting a bit nicer + automatically creating the pull request but this is iteration 1 :)